webpack.prod.config.js

```diff
@@ -41,18 +48,10 @@ module.exports = {
     ],
   },
   externalsPresets: { node: true },
-  externals: [
-    nodeExternals({
-      allowlist: [
-        'dompurify',
-        '/^@fullcalendar/',
-        'marked',
-        'ol',
-        'pluralize',
-        'recharts',
```

Comment:
I think all of these should be added as peerDependencies since we are no longer bundling them.